package.json

```diff
@@ -82,7 +83,8 @@
     "extends": [
       "prettier",
       "plugin:@typescript-eslint/recommended",
-      "standard"
+      "standard",
+      "plugin:react-hooks/recommended"
```

Comment:
Can you move this package to [boilerplate/package.json](https://github.com/infinitered/ignite/blob/master/boilerplate/package.json) instead?

This top level package.json is for the cli, where as the boilerplate folder is for the app content.